### ISSUE TITLE
Rails 5 before/after_filter deprecation warnings

### ISSUE THREAD

**Issue by terracatta** [assaf/vanity#314]
The master branch of vanity will throw the following deprecation warnings on a Rails 5.x application.

```
DEPRECATION WARNING: around_filter is deprecated and will be removed in Rails 5.1. Use around_action instead. (called from use_vanity at /Users/<snip>/.rvm/gems/ruby-2.3.1/bundler/gems/vanity-6d55564d417b/lib/vanity/frameworks/rails.rb:47)
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from use_vanity at /Users/<snip>/.rvm/gems/ruby-2.3.1/bundler/gems/vanity-6d55564d417b/lib/vanity/frameworks/rails.rb:48)
DEPRECATION WARNING: before_filter is deprecated and will be removed in Rails 5.1. Use before_action instead. (called from use_vanity at /Users/<snip>/.rvm/gems/ruby-2.3.1/bundler/gems/vanity-6d55564d417b/lib/vanity/frameworks/rails.rb:49)
DEPRECATION WARNING: after_filter is deprecated and will be removed in Rails 5.1. Use after_action instead. (called from use_vanity at /Users/<snip>/.rvm/gems/ruby-2.3.1/bundler/gems/vanity-6d55564d417b/lib/vanity/frameworks/rails.rb:50)
```
Offending lines are here:
https://github.com/assaf/vanity/blob/master/lib/vanity/frameworks/rails.rb#L47-L50

**Comment by mateuszgorniak** [comment]
Is it possible to merge it ASAP? Rails 5.1 is almost ready, It would be awesome to upgrade it soon :-)

**Comment by phillbaker** [comment]
Fixed by #326.
